app/views/assignments/index.html.erb

```diff
@@ -1,49 +1,59 @@
+
+
 <h1><%= @current_office.name %> Kitchen Schedule</h1>
 
-<table>
-<tr>
-  <th>
-    Dates
-  </th>
-  <th>
-    Employees
-  </th>
-</tr>
 
-<% @weeks.each do |week| %>
-  <tr>
-    <td>
-      <%= print_date(week) %>
-    </td>
+<table class="table table-sm table-borderless">
+  <thead class="bg-dark table-dark">
+    <tr>
+      <th>
+        Dates
+      </th>
+      <th>
+        Employees
+      </th>
+
+      <th></th>
+  
+      <% if can_edit_weeks? %>
+        <th></th>
+      <% end %>
+      </tr>
+    </thead>
+  <tbody>
+    <% @weeks.each do |week| %>
+      <tr>
+        <td>
+          <%= print_date(week) %>
+        </td>
     
-    <% week_assignments = week.employees.order(:first_name) %>
-    <% week_assignments.each do |employee| %>
-      <td>
-        <%= "#{employee.first_name}" %>
-      </td>
-    <% end %>
+        <% week_assignments = week.employees.order(:first_name) %>
+        <% week_assignments.each do |employee| %>
+          <td>
+            <%= "#{employee.first_name}" %>
+          </td>
+        <% end %>
     
-    <% if week_assignments.count != 2 %>
-      <% (2 - week_assignments.count).times do %>
-        <td><b>EMPTY</b></td>
-      <% end %>
-    <%end%>
+        <% if week_assignments.count != 2 %>
+          <% (2 - week_assignments.count).times do %>
```

Comment:
Lots of dependency on 2 assignments per week threaded throughout the app. I think it's fine but I would be using a constant like `Week::ASSIGNMENTS_PER_WEEK` or application configuration variable.

Reply:
I already had a fully assigned method for the week model, but I guess I forgot to make sure i actually use it here.